llvm/lib/Target/X86/X86FlagsCopyLowering.cpp

```diff
@@ -970,3 +977,29 @@ void X86FlagsCopyLoweringPass::rewriteSetCC(MachineBasicBlock &TestMBB,
 
   SetCCI.eraseFromParent();
 }
+
+void X86FlagsCopyLoweringPass::rewriteCCMP(MachineBasicBlock &TestMBB,
```

Comment:
This look very similar to `rewriteCMov`, can we merge them together?

Reply:
Most of the `rewrite*` functions can be merged together, including `rewriteCMov`. I will do it after this PR.